### PR TITLE
ncm-autofs: switch to 00-tqu and buildtools 1.52

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: ncm-autofs/src/main/perl/autofs.pm [hunk 1]
```diff
@@ -1,19 +1,66 @@
...
+
+=head2 Scenario 2 : Configuration with nlist() usage
+
+    prefix '/software/components/autofs';
+    'preserveMaster' = false;
 
+        'maps', nlist(
+
+            'misc', nlist(
+    prefix '/software/components/autofs/maps/misc';
```
Comment (ned21) [inline ncm-autofs/src/main/perl/autofs.pm]: This line seems out of place

**Comment by stdweird** [comment]
@ned21 done

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
retest this please

**Pull request merged**